cultura.html

```diff
@@ -14,6 +14,7 @@ <h2>Calidad</h2>
   <li>Revisamos nuestro código vía Pull Requests. Nada pasa a master sin que haya sido aprobado por al menos otro miembro del equipo.</li>
   <li>Testamos nuestro código, o justificamos por qué en un caso concreto no es necesario hacerlo.</li>
   <li>Siempre tratamos de dejar el código mejor que cuando lo encontramos por primera vez.</li>
+  <li>Siempre dejamos el código legible para el resto del equipo y nuestros yos del futuro. Generalmente haremos caso a las herramientas automáticas, que están basadas en convenciones con sentido y ampliamente extendidas, pero lo ignoraremos cuando su versión deje el código menos legible.</li>
```

Comment:
Maybe we can merge the previous line with this, now seems a bit redundant.

Reply:
Hmmm... I think "dejar el código mejor" is wider than "dejamos el código legible". I'd say the first line includes quick refactors, for example.